src/pgduckdb_planner.cpp

```diff
@@ -9,7 +9,7 @@
 extern "C" {
 #include "postgres.h"
 #include "access/xact.h"
-#include "catalog/pg_type.h"
+#include "catalog/pg_type.h" 
```

Comment:
accidental space it seems. Did you install editorconfig? https://editorconfig.org/ That should make sure you adhere to the whitespace standards across all files.